fuzz/fuzz_targets/encode_decode.rs

```diff
@@ -13,26 +15,22 @@ fn do_test(data: &[u8]) {
         return;
     }
 
-    let hrp = String::from_utf8_lossy(&data[1..hrp_end])
-        .to_lowercase()
-        .to_string();
+    let hrp = String::from_utf8_lossy(&data[1..hrp_end]).to_lowercase().to_string();
```

Comment:
In 179868360b76d5d9a8d806dd5721d51d84e84b48:

Did rustfmt make all these changes to this fuzz unit? They're a little distracting.

I have a bunch of comments about this fuzztest but since it already exists it's probably better to deal with it in a separate PR..

Reply:
Sorry about that, I removed the rustfmt changes. PR still has changes to this fuzz test, I took your comments on the other one and applied it to this test. We have to touch this fuzz test in the PR because this PR changes the API.